packages/amfora-bin/amfora-bin.pacscript

```diff
@@ -0,0 +1,25 @@
+name="amfora-bin"
+pkgname="amfora"
+version="1.9.2"
+pkgdir="${STOWDIR}/${name}"
+gives="${pkgname}"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-deb ${pkgname}-app"
+description="Amfora aims to be the best looking Gemini client with the most features"
+hash="a468f97f0e3fa1d69868980fd0e5893388ccd9c849874d26e4d0b426fd6bff3e"
+url="https://github.com/makeworld-the-better-one/amfora/releases/download/v${version}/amfora_${version}_linux_64-bit"
+maintainer="Henryws <hwengerstickel@pm.me>"
+
+prepare() {
+  wget -q https://roboticoverlords.org/amfora.png
+  wget -q https://raw.githubusercontent.com/makeworld-the-better-one/amfora/master/amfora.desktop
+}
+
+build() {
+  true
+}
+
+install() {
+  sudo install -Dm644 amfora.png "${pkgdir}/usr/share/pixmaps/amfora.png"
+  sudo install -Dm644 amfora.desktop "${pkgdir}/usr/share/applications/amfora.desktop"
+  sudo install -Dm755 amfora_"${version}"_linux_64-bit "${pkgdir}/usr/bin/${pkgname}"
```

Comment:
```suggestion
  mv "amfora_"${version}"_linux_64-bit" "amfora"
  sudo install -Dm755 "amfora" "${pkgdir}/usr/bin/${pkgname}"
```

Reply:
Any real reason to do this?

Reply:
![image](https://user-images.githubusercontent.com/70437021/148310753-00a9cf24-1987-4305-b2e1-5803a1df4194.png)
the .desktop calls `amfora`, not `amfora_"${version}"_linux_64-bit`

Reply:
also it'll be easier to call with terminal

Reply:
Yes but we are using the `install` command to install the long binary to the name `amfora`

Reply:
oh wait I didn't see pkgname